assets/js/utils/dashboardUtils.ts

```diff
@@ -0,0 +1,61 @@
+import axiosWithInterceptor from "../services/index";
+
+export type EventForm = {
+  title: string;
+  description: string;
+  start_date: string;
+  presenters: string[];
+};
+
+export type WebinarInfo = {
+  title: string;
+  description: string;
+  presenters: string[];
+  start_date: Date;
```

Comment:
As this is exported type this key should be written in camelCase.

Reply:
Fixed (kind of).